lesson2.html

```diff
@@ -86,8 +86,8 @@ <h2>Lesson 2</h2>
 						}
 						</code></pre>
 						<p>Lines 1 and 5 define a <strong>class</strong></p>
-						<p>A class lets you bundle related functionality into one file, and use it in another</p>
-						<p>All functionality must be defined inside a class, only <strong>package</strong> and <strong>import</strong> go outside.</p>
+                        <p>A class lets you bundle related functionality into one file, and can then be used in another class.</p>
+                        <p>All code must be defined inside a class!</p>
```

Comment:
do we want to keep the import and package comment as a <small> on this page?

Reply:
I think maybe it's worth creating a separate slide to explain what both of those are. I think in this context, without them having been introduced, it was a little confusing.